README.md

```diff
@@ -77,8 +77,7 @@ import { Provider } from 'react-redux';
 render(
   <Provider store={store}>
     <MyRootComponent />
-  </Provider>,
-  rootEl,
```

Comment:
The second argument is the container in which to render, so that is a necessary argument. What was the reason for removing?